src/main/proto/marathon.proto

```diff
@@ -141,6 +141,22 @@ message MarathonTask {
     repeated string local_volume_ids = 1;
     required State state = 2;
   }
+  enum MarathonTaskStatus {
```

Comment:
pretty sure that I know what most of these mean. that said, docs would be appreciated